### ISSUE TITLE
ALOps Extension API Step - displays error although all apps were published

### ISSUE THREAD

**Issue by mrabus** [HodorNV/ALOps#200]
**Describe the bug**
We publish several apps through the API and although publishing works fine, the step displays eight errors. 

**the used yaml**
I have a rather simple release pipeline to deploy 8 apps to my local on prem BC16 testing environment: 
![image](https://user-images.githubusercontent.com/2485163/93803084-a0902b80-fc44-11ea-968e-39c094b4006e.png)

**the output**
The pipeline is executed and works fine so far, but it deploys errors in the deployment step: 
![image](https://user-images.githubusercontent.com/2485163/93803194-c5849e80-fc44-11ea-8e38-b744015f9921.png)

```
020-09-21T17:25:35.6677969Z ##[command]Invoke-RestMethod -Method Get -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensionDeploymentStatus'
2020-09-21T17:25:36.6736487Z *** Deployment Status:
2020-09-21T17:25:36.6736917Z 
2020-09-21T17:25:36.6737282Z 
2020-09-21T17:25:36.6739829Z operationID : 64ea9265-a5d9-492a-a9e9-b7395a667401
2020-09-21T17:25:36.6740492Z name        : easyPDF
2020-09-21T17:25:36.6741166Z publisher   : B.i.TEAM Gesellschaft für Softwareberatung mbH
2020-09-21T17:25:36.6741805Z appVersion  : 1.0.202037.614
2020-09-21T17:25:36.6742502Z status      : Unknown
2020-09-21T17:25:36.6744050Z startedOn   : 2020-09-21T17:25:35.603Z
2020-09-21T17:25:36.6744574Z 
2020-09-21T17:25:36.6744836Z 
2020-09-21T17:25:36.6744980Z 
2020-09-21T17:25:36.6779303Z ##[error]App publish failed. Check the 'Extension Deployment Status' page in the Business Central client.
2020-09-21T17:25:36.6789570Z *** Publish App [C:\agent\_work\r1\a\_BC_General\XXXXX\B.i.TEAM Gesellschaft für Softwareberatung mbH_XXXXX General_1.0.202038.673.app]
2020-09-21T17:25:36.6790864Z *** Upload extension [C:\agent\_work\r1\a\_BC_General\XXXXX\B.i.TEAM Gesellschaft für Softwareberatung mbH_XXXXX General_1.0.202038.673.app]
2020-09-21T17:25:36.6791689Z *** Get App Info
2020-09-21T17:25:36.6792641Z  * App.ID        = f0387a01-4ad3-4600-9cd7-14bf0f3c85d8
2020-09-21T17:25:36.6793302Z  * App.Name      = XXXXX General
2020-09-21T17:25:36.6793968Z  * App.Publisher = B.i.TEAM Gesellschaft für Softwareberatung mbH
2020-09-21T17:25:36.6794636Z  * App.Version   = 1.0.202038.673
2020-09-21T17:25:36.6795172Z *** Get Extension status
2020-09-21T17:25:36.6796357Z ##[command]Invoke-RestMethod -Method Get -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensions'
2020-09-21T17:25:36.6797583Z *** Get Extension Deployment status
2020-09-21T17:25:36.6798841Z ##[command]Invoke-RestMethod -Method Get -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensionDeploymentStatus'
2020-09-21T17:25:36.6800960Z ##[command]Invoke-RestMethod -Method Patch -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensionUpload(0)/content'
2020-09-21T17:26:47.3465758Z 
2020-09-21T17:26:47.3481242Z *** Poll Deployment status
2020-09-21T17:26:47.3499695Z ##[command]Invoke-RestMethod -Method Get -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensionDeploymentStatus'
2020-09-21T17:26:47.4421788Z *** Deployment Status:
2020-09-21T17:26:47.4426863Z 
2020-09-21T17:26:47.4430702Z 
2020-09-21T17:26:47.4441820Z operationID : ea163bed-7aab-4cbb-ab0d-c6738071d99c
2020-09-21T17:26:47.4442361Z name        : XXXXX General
2020-09-21T17:26:47.4444546Z publisher   : B.i.TEAM Gesellschaft für Softwareberatung mbH
2020-09-21T17:26:47.4445634Z appVersion  : 1.0.202038.673
2020-09-21T17:26:47.4446015Z status      : Unknown
2020-09-21T17:26:47.4446899Z startedOn   : 2020-09-21T17:26:47.293Z
2020-09-21T17:26:47.4447224Z 
2020-09-21T17:26:47.4449985Z 
2020-09-21T17:26:47.4452385Z 
2020-09-21T17:26:47.4472799Z ##[error]App publish failed. Check the 'Extension Deployment Status' page in the Business Central client.
2020-09-21T17:26:47.4491008Z *** Publish App [C:\agent\_work\r1\a\_BC_VNT\XXXXX\B.i.TEAM Gesellschaft für Softwareberatung mbH_VNTBC_1.0.202037.615.app]
2020-09-21T17:26:47.4512620Z *** Upload extension [C:\agent\_work\r1\a\_BC_VNT\XXXXX\B.i.TEAM Gesellschaft für Softwareberatung mbH_VNTBC_1.0.202037.615.app]
2020-09-21T17:26:47.4535888Z *** Get App Info
2020-09-21T17:26:47.4578124Z  * App.ID        = 2ecd17c5-17d8-4526-a859-1181368325d8
2020-09-21T17:26:47.4597023Z  * App.Name      = VNTBC
2020-09-21T17:26:47.4615473Z  * App.Publisher = B.i.TEAM Gesellschaft für Softwareberatung mbH
2020-09-21T17:26:47.4633784Z  * App.Version   = 1.0.202037.615
2020-09-21T17:26:47.4650684Z *** Get Extension status
2020-09-21T17:26:47.4672834Z ##[command]Invoke-RestMethod -Method Get -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensions'
2020-09-21T17:26:47.4993499Z *** Get Extension Deployment status
2020-09-21T17:26:47.5011685Z ##[command]Invoke-RestMethod -Method Get -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensionDeploymentStatus'
2020-09-21T17:26:47.5266416Z ##[command]Invoke-RestMethod -Method Patch -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensionUpload(0)/content'
2020-09-21T17:27:07.7756334Z 
2020-09-21T17:27:07.7776284Z *** Poll Deployment status
2020-09-21T17:27:07.7799210Z ##[command]Invoke-RestMethod -Method Get -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensionDeploymentStatus'
2020-09-21T17:27:08.1028245Z *** Deployment Status:
2020-09-21T17:27:08.1033420Z 
2020-09-21T17:27:08.1036800Z 
2020-09-21T17:27:08.1048167Z operationID : cc6a9c9e-5253-44d5-a085-84c851d94757
2020-09-21T17:27:08.1053565Z name        : VNTBC
2020-09-21T17:27:08.1058119Z publisher   : B.i.TEAM Gesellschaft für Softwareberatung mbH
2020-09-21T17:27:08.1060402Z appVersion  : 1.0.202037.615
2020-09-21T17:27:08.1060785Z status      : Unknown
2020-09-21T17:27:08.1064003Z startedOn   : 2020-09-21T17:27:07.767Z
2020-09-21T17:27:08.1064615Z 
2020-09-21T17:27:08.1064916Z 
2020-09-21T17:27:08.1067036Z 
2020-09-21T17:27:08.1087448Z ##[error]App publish failed. Check the 'Extension Deployment Status' page in the Business Central client.
2020-09-21T17:27:08.1105110Z *** Publish App [C:\agent\_work\r1\a\_opplus\OPPlus.app]
2020-09-21T17:27:08.1125668Z *** Upload extension [C:\agent\_work\r1\a\_opplus\OPPlus.app]
2020-09-21T17:27:08.1148758Z *** Get App Info
2020-09-21T17:27:09.1408658Z  * App.ID        = 8f01b742-ad1b-48b0-9369-32ed713dd3e6
2020-09-21T17:27:09.1409498Z  * App.Name      = OPplus Extension
2020-09-21T17:27:09.1410151Z  * App.Publisher = gbedv GmbH & Co. KG
2020-09-21T17:27:09.1412907Z  * App.Version   = 16.0.200429.0
2020-09-21T17:27:09.1419847Z *** Get Extension status
2020-09-21T17:27:09.1422033Z ##[command]Invoke-RestMethod -Method Get -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensions'
2020-09-21T17:27:09.1423449Z *** Get Extension Deployment status
2020-09-21T17:27:09.1424682Z ##[command]Invoke-RestMethod -Method Get -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensionDeploymentStatus'
2020-09-21T17:27:09.1426671Z ##[command]Invoke-RestMethod -Method Patch -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensionUpload(0)/content'
2020-09-21T17:28:53.1452502Z 
2020-09-21T17:28:53.1470028Z *** Poll Deployment status
2020-09-21T17:28:53.1489011Z ##[command]Invoke-RestMethod -Method Get -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensionDeploymentStatus'
2020-09-21T17:28:54.6381854Z *** Deployment Status:
2020-09-21T17:28:54.6386627Z 
2020-09-21T17:28:54.6389653Z 
2020-09-21T17:28:54.6397394Z operationID : 5d4799a5-57b1-438f-ae90-9dd64f6380e6
2020-09-21T17:28:54.6399054Z name        : OPplus Extension
2020-09-21T17:28:54.6401064Z publisher   : gbedv GmbH & Co. KG
2020-09-21T17:28:54.6401534Z appVersion  : 16.0.200429.0
2020-09-21T17:28:54.6404802Z status      : Unknown
2020-09-21T17:28:54.6405293Z startedOn   : 2020-09-21T17:28:52.897Z
2020-09-21T17:28:54.6406686Z 
2020-09-21T17:28:54.6407029Z 
2020-09-21T17:28:54.6409777Z 
2020-09-21T17:28:54.6426709Z ##[error]App publish failed. Check the 'Extension Deployment Status' page in the Business Central client.
2020-09-21T17:28:54.6441896Z *** Publish App [C:\agent\_work\r1\a\_BC_Shipping\XXXXX\B.i.TEAM Gesellschaft für Softwareberatung mbH_XXXXX Shipping Module_1.0.202037.617.app]
2020-09-21T17:28:54.6459370Z *** Upload extension [C:\agent\_work\r1\a\_BC_Shipping\XXXXX\B.i.TEAM Gesellschaft für Softwareberatung mbH_XXXXX Shipping Module_1.0.202037.617.app]
2020-09-21T17:28:54.6479398Z *** Get App Info
2020-09-21T17:28:54.6519834Z  * App.ID        = 57979a35-742f-46de-8d60-59b49dec2151
2020-09-21T17:28:54.6534960Z  * App.Name      = XXXXX Shipping Module
2020-09-21T17:28:54.6549707Z  * App.Publisher = B.i.TEAM Gesellschaft für Softwareberatung mbH
2020-09-21T17:28:54.6564680Z  * App.Version   = 1.0.202037.617
2020-09-21T17:28:54.6578189Z *** Get Extension status
2020-09-21T17:28:54.6596564Z ##[command]Invoke-RestMethod -Method Get -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensions'
2020-09-21T17:28:55.4182131Z *** Get Extension Deployment status
2020-09-21T17:28:55.4200117Z ##[command]Invoke-RestMethod -Method Get -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensionDeploymentStatus'
2020-09-21T17:28:56.5484851Z ##[command]Invoke-RestMethod -Method Patch -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensionUpload(0)/content'
2020-09-21T17:29:17.5361045Z 
2020-09-21T17:29:17.5380353Z *** Poll Deployment status
2020-09-21T17:29:17.5404066Z ##[command]Invoke-RestMethod -Method Get -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensionDeploymentStatus'
2020-09-21T17:29:17.6123573Z *** Deployment Status:
2020-09-21T17:29:17.6129010Z 
2020-09-21T17:29:17.6132243Z 
2020-09-21T17:29:17.6142249Z operationID : e2439899-9c45-4d82-9dc8-fa8e62190405
2020-09-21T17:29:17.6142770Z name        : XXXXX Shipping Module
2020-09-21T17:29:17.6146058Z publisher   : B.i.TEAM Gesellschaft für Softwareberatung mbH
2020-09-21T17:29:17.6146591Z appVersion  : 1.0.202037.617
2020-09-21T17:29:17.6149608Z status      : Unknown
2020-09-21T17:29:17.6150037Z startedOn   : 2020-09-21T17:29:17.53Z
2020-09-21T17:29:17.6151509Z 
2020-09-21T17:29:17.6152161Z 
2020-09-21T17:29:17.6153362Z 
2020-09-21T17:29:17.6174478Z ##[error]App publish failed. Check the 'Extension Deployment Status' page in the Business Central client.
2020-09-21T17:29:17.6196163Z *** Publish App [C:\agent\_work\r1\a\_BC_MCS_Interface\XXXXX\B.i.TEAM Gesellschaft für Softwareberatung mbH_XXXXX MCS-Interface_1.0.202037.619.app]
2020-09-21T17:29:17.6221559Z *** Upload extension [C:\agent\_work\r1\a\_BC_MCS_Interface\XXXXX\B.i.TEAM Gesellschaft für Softwareberatung mbH_XXXXX MCS-Interface_1.0.202037.619.app]
2020-09-21T17:29:17.6245005Z *** Get App Info
2020-09-21T17:29:17.6277684Z  * App.ID        = 867d7279-1b26-44a1-9788-587b2aa7da39
2020-09-21T17:29:17.6296073Z  * App.Name      = XXXXX MCS-Interface
2020-09-21T17:29:17.6314614Z  * App.Publisher = B.i.TEAM Gesellschaft für Softwareberatung mbH
2020-09-21T17:29:17.6332946Z  * App.Version   = 1.0.202037.619
2020-09-21T17:29:17.6350038Z *** Get Extension status
2020-09-21T17:29:17.6372269Z ##[command]Invoke-RestMethod -Method Get -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensions'
2020-09-21T17:29:17.6782461Z *** Get Extension Deployment status
2020-09-21T17:29:17.6804568Z ##[command]Invoke-RestMethod -Method Get -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensionDeploymentStatus'
2020-09-21T17:29:17.7164323Z ##[command]Invoke-RestMethod -Method Patch -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensionUpload(0)/content'
2020-09-21T17:29:23.6861529Z 
2020-09-21T17:29:23.6883632Z *** Poll Deployment status
2020-09-21T17:29:23.6907202Z ##[command]Invoke-RestMethod -Method Get -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensionDeploymentStatus'
2020-09-21T17:29:24.4997917Z *** Deployment Status:
2020-09-21T17:29:24.5003040Z 
2020-09-21T17:29:24.5006002Z 
2020-09-21T17:29:24.5017313Z operationID : 4270d771-9a61-4e2a-8fa6-9a18a25e22b5
2020-09-21T17:29:24.5018149Z name        : XXXXX MCS-Interface
2020-09-21T17:29:24.5021329Z publisher   : B.i.TEAM Gesellschaft für Softwareberatung mbH
2020-09-21T17:29:24.5021800Z appVersion  : 1.0.202037.619
2020-09-21T17:29:24.5024227Z status      : Unknown
2020-09-21T17:29:24.5026736Z startedOn   : 2020-09-21T17:29:23.643Z
2020-09-21T17:29:24.5027157Z 
2020-09-21T17:29:24.5027313Z 
2020-09-21T17:29:24.5027451Z 
2020-09-21T17:29:24.5046419Z ##[error]App publish failed. Check the 'Extension Deployment Status' page in the Business Central client.
2020-09-21T17:29:24.5062669Z *** Publish App [C:\agent\_work\r1\a\_BC_OPplus_Extension\XXXXX\B.i.TEAM Gesellschaft für Softwareberatung mbH_XXXXX OPplus Extension_1.0.202037.621.app]
2020-09-21T17:29:24.5083779Z *** Upload extension [C:\agent\_work\r1\a\_BC_OPplus_Extension\XXXXX\B.i.TEAM Gesellschaft für Softwareberatung mbH_XXXXX OPplus Extension_1.0.202037.621.app]
2020-09-21T17:29:24.5107411Z *** Get App Info
2020-09-21T17:29:24.5139910Z  * App.ID        = e9017eeb-d60a-4999-9c9e-f401fa54f837
2020-09-21T17:29:24.5158810Z  * App.Name      = XXXXX OPplus Extension
2020-09-21T17:29:24.5176005Z  * App.Publisher = B.i.TEAM Gesellschaft für Softwareberatung mbH
2020-09-21T17:29:24.5200857Z  * App.Version   = 1.0.202037.621
2020-09-21T17:29:24.5218195Z *** Get Extension status
2020-09-21T17:29:24.5239475Z ##[command]Invoke-RestMethod -Method Get -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensions'
2020-09-21T17:29:24.7133046Z *** Get Extension Deployment status
2020-09-21T17:29:24.7158759Z ##[command]Invoke-RestMethod -Method Get -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensionDeploymentStatus'
2020-09-21T17:29:24.8098157Z ##[command]Invoke-RestMethod -Method Patch -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensionUpload(0)/content'
2020-09-21T17:29:34.6836310Z 
2020-09-21T17:29:34.6854171Z *** Poll Deployment status
2020-09-21T17:29:34.6876330Z ##[command]Invoke-RestMethod -Method Get -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensionDeploymentStatus'
2020-09-21T17:29:35.0838506Z *** Deployment Status:
2020-09-21T17:29:35.0843200Z 
2020-09-21T17:29:35.0846251Z 
2020-09-21T17:29:35.0857275Z operationID : d896b551-46e7-4f01-be01-2d79c0825702
2020-09-21T17:29:35.0858129Z name        : XXXXX OPplus Extension
2020-09-21T17:29:35.0859275Z publisher   : B.i.TEAM Gesellschaft für Softwareberatung mbH
2020-09-21T17:29:35.0860269Z appVersion  : 1.0.202037.621
2020-09-21T17:29:35.0863932Z status      : Unknown
2020-09-21T17:29:35.0865060Z startedOn   : 2020-09-21T17:29:34.68Z
2020-09-21T17:29:35.0865824Z 
2020-09-21T17:29:35.0867793Z 
2020-09-21T17:29:35.0868518Z 
2020-09-21T17:29:35.0890200Z ##[error]App publish failed. Check the 'Extension Deployment Status' page in the Business Central client.
2020-09-21T17:29:35.0905847Z *** Publish App [C:\agent\_work\r1\a\_BC_Documents\XXXXX\B.i.TEAM Gesellschaft für Softwareberatung mbH_BC_Documents_1.0.202038.689.app]
2020-09-21T17:29:35.0926096Z *** Upload extension [C:\agent\_work\r1\a\_BC_Documents\XXXXX\B.i.TEAM Gesellschaft für Softwareberatung mbH_BC_Documents_1.0.202038.689.app]
2020-09-21T17:29:35.0950232Z *** Get App Info
2020-09-21T17:29:35.0984168Z  * App.ID        = 0f30a753-891c-45f8-ae6c-013aafe4a114
2020-09-21T17:29:35.1002992Z  * App.Name      = BC_Documents
2020-09-21T17:29:35.1021236Z  * App.Publisher = B.i.TEAM Gesellschaft für Softwareberatung mbH
2020-09-21T17:29:35.1040503Z  * App.Version   = 1.0.202038.689
2020-09-21T17:29:35.1069257Z *** Get Extension status
2020-09-21T17:29:35.1091582Z ##[command]Invoke-RestMethod -Method Get -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensions'
2020-09-21T17:29:35.2938133Z *** Get Extension Deployment status
2020-09-21T17:29:35.2959811Z ##[command]Invoke-RestMethod -Method Get -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensionDeploymentStatus'
2020-09-21T17:29:35.5463113Z ##[command]Invoke-RestMethod -Method Patch -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensionUpload(0)/content'
2020-09-21T17:29:43.6493710Z 
2020-09-21T17:29:43.6510625Z *** Poll Deployment status
2020-09-21T17:29:43.6528837Z ##[command]Invoke-RestMethod -Method Get -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensionDeploymentStatus'
2020-09-21T17:29:43.7794953Z *** Deployment Status:
2020-09-21T17:29:43.7800213Z 
2020-09-21T17:29:43.7802997Z 
2020-09-21T17:29:43.7809852Z operationID : 4b5fd0f6-c615-4d0f-ba62-4abd974bab2b
2020-09-21T17:29:43.7810593Z name        : BC_Documents
2020-09-21T17:29:43.7812456Z publisher   : B.i.TEAM Gesellschaft für Softwareberatung mbH
2020-09-21T17:29:43.7813187Z appVersion  : 1.0.202038.689
2020-09-21T17:29:43.7813740Z status      : Unknown
2020-09-21T17:29:43.7814160Z startedOn   : 2020-09-21T17:29:43.617Z
2020-09-21T17:29:43.7814430Z 
2020-09-21T17:29:43.7814614Z 
2020-09-21T17:29:43.7816218Z 
2020-09-21T17:29:43.7833025Z ##[error]App publish failed. Check the 'Extension Deployment Status' page in the Business Central client.
2020-09-21T17:29:43.7875631Z ##[command]Invoke-RestMethod -Method Get -Uri 'http://bits123.bit.intra:7068/XXXXXdeploy_bc160_14238/api/microsoft/automation/beta/companies(17b98b75-5fb7-ea11-bb3d-001dd8b767da)/extensionDeploymentStatus'
2020-09-21T17:29:44.2389319Z 
2020-09-21T17:29:44.2504658Z name                   publisher                                      appVersion     operationType schedule status    s
2020-09-21T17:29:44.2505952Z                                                                                                                       t
2020-09-21T17:29:44.2506999Z                                                                                                                       a
2020-09-21T17:29:44.2508066Z                                                                                                                       r
2020-09-21T17:29:44.2509685Z                                                                                                                       t
2020-09-21T17:29:44.2510825Z                                                                                                                       e
2020-09-21T17:29:44.2511873Z                                                                                                                       d
2020-09-21T17:29:44.2512907Z                                                                                                                       O
2020-09-21T17:29:44.2513944Z                                                                                                                       n
2020-09-21T17:29:44.2535654Z ----                   ---------                                      ----------     ------------- -------- ------    -
2020-09-21T17:29:44.2550727Z BC_Documents           B.i.TEAM Gesellschaft für Softwareberatung mbH 1.0.202038.689 Upload        Sofort   Unknown   2
2020-09-21T17:29:44.2566714Z easyPDF                B.i.TEAM Gesellschaft für Softwareberatung mbH 1.0.202037.614 Upload        Sofort   Unknown   2
2020-09-21T17:29:44.2581639Z XXXXX General          B.i.TEAM Gesellschaft für Softwareberatung mbH 1.0.202038.673 Upload        Sofort   Unknown   2
2020-09-21T17:29:44.2596475Z XXXXX MCS-Interface    B.i.TEAM Gesellschaft für Softwareberatung mbH 1.0.202037.619 Upload        Sofort   Unknown   2
2020-09-21T17:29:44.2611518Z XXXXX OPplus Extension B.i.TEAM Gesellschaft für Softwareberatung mbH 1.0.202037.621 Upload        Sofort   Unknown   2
2020-09-21T17:29:44.2626474Z XXXXX Shipping Module  B.i.TEAM Gesellschaft für Softwareberatung mbH 1.0.202037.617 Upload        Sofort   Unknown   2
2020-09-21T17:29:44.2641589Z OPplus Extension       gbedv GmbH & Co. KG                            16.0.200429.0  Upload        Sofort   Completed 2
2020-09-21T17:29:44.2656723Z VNTBC                  B.i.TEAM Gesellschaft für Softwareberatung mbH 1.0.202037.615 Upload        Sofort   Unknown   2
2020-09-21T17:29:44.2657480Z 
2020-09-21T17:29:44.2659360Z 
2020-09-21T17:29:44.2778965Z *** Cleanup VSTS Environment: True
2020-09-21T17:29:46.5856318Z ##[section]Finishing: ALOps Extension API

```

**Expected behavior**
The apps were successfully deployed but the status in the API is unknown. I think the issue is rather the BC API than ALOps but maybe you have an idea what we can about it. 
I would expect the step to display no errors if the apps have been deployed successfully

**Comment by waldo1001** [comment]
It's a bug indeed - we are aware - it's on the list to be fixed!

**Comment by AdminHodor** [comment]
Dear @mrabus,

We just released v1.435.1856 which should tackle this issue.
Please note it could take a couple of pipeline runs for DevOps to pickup the new version.

Kind regards,
